snewpdag/data/fd2210.csv

```diff
@@ -0,0 +1,23 @@
+"Control","Pass",,"'line': 100"
+,,,
+"idcopy","Copy","Control","'cp':(('_id','burst_id'),)"
+,,,
+"Dts","DiffTimes","idcopy","'detector_location':'snewpdag/data/detector_location_zerobias.csv'"
+"Dts-out","Pass","Dts","'line':1, 'dump':1"
+,,,
+"Diff","DiffPointing","Dts","'detector_location':'snewpdag/data/detector_location_zerobias.csv', 'nside':32, 'min_dts':3"
+"Diff-out","Pass","Diff","'line':1, 'dump':1"
+,,,
+"chi2map","renderers.Mollview","Diff","'in_field':'map', 'title':'DiffPointing chi2', 'units':'chi2', 'coord':['C'], 'filename':'output/fd2210-{0}-{2}.png'"
+"chi2fits","renderers.FitsSkymap","Diff","'in_field':'map', 'filename':'output/fd2210-{0}-{2}.fits'"
+,,,
+"prob","Chi2Prob","Diff","'in_field':'map', 'out_field':'probmap'"
+"prob-out","Pass","prob","'line':1, 'dump':1"
+"probmap","renderers.Mollview","prob","'in_field':'probmap', 'title':'DiffPointing probability', 'units':'probability', 'coord':['C'], 'filename':'output/fd2210-{0}-{2}.png'"
+"fits","renderers.FitsSkymap","prob","'in_field':'probmap', 'filename':'output/fd2210-{0}-{2}.fits'"
+,,,
+"conf","Chi2CL","Diff","'in_field':'map', 'out_field':'clmap'"
+"conf-out","Pass","conf","'line':1, 'dump':1"
+"confmap","renderers.Mollview","conf","'in_field':'clmap', 'title':'DiffPointing CL', 'units':'CL', 'range':(0,1), 'coord':['C'], 'filename':'output/fd2210-{0}-{2}.png'"
+"conffits","renderers.FitsSkymap","conf","'in_field':'clmap', 'filename':'output/fd2210-{0}-{2}.fits'"
+,,,
```

Comment:
Minor suggestion:

As "_id" already contains the date, I would avoid having "fd2210" in the output filename, maybe just "fd-out.."?

Reply:
I kept the fd2210 there so it was obvious which configuration script was used.